WhisperAPI/WhisperAPI.Tests/Integration/SuggestionsControllerTest.cs

```diff
@@ -248,7 +248,7 @@ public void When_getting_suggestions_with_customer_sending_uri_then_suggestion_i
         [Test]
         public void When_getting_suggestions_with_more_attribute_then_required_then_returns_correctly()
         {
-            var jsonSearchQuery = "{\"chatkey\": \"aecaa8db-abc8-4ac9-aa8d-87987da2dbb0\",\"Query\": \"Need help with CoveoSearch API\",\"Type\": 1,\"command\": \"sudo ls\"}";
+            var jsonSearchQuery = "{\"chatkey\": \"aecaa8db-abc8-4ac9-aa8d-87987da2dbb0\",\"Query\": \"Need help with CoveoSearch API\",\"Type\": 1,\"command\": \"sudo ls\",\"maxDocuments\": \"10\",\"maxQuestions\": \"10\" }";
```

Comment:
No, Because we are testing that When_getting_suggestions_with_more_attribute_then_required_then_returns_correctly ;)
So it have to come from a deserialized object